images/base/Dockerfile

```diff
@@ -13,5 +13,5 @@ RUN INSTALL_PKGS="bsdtar ceph-common device-mapper device-mapper-persistent-data
     mkdir -p /var/lib/origin
 
 LABEL io.openshift.tags="openshift,base" \
-      io.k8s.display-name="OpenShift Origin CentOS 7 Base" \
```

Comment:
Based on https://github.com/openshift/origin/pull/13642#discussion_r110762845 I figured we can